plugins/woocommerce-admin/client/customize-store/assembler-hub/style.scss

```diff
@@ -587,7 +586,7 @@ body.woocommerce-assembler {
 
 	.customize-your-store-edit-site-resizable-frame__inner-content {
 		height: 100%;
-		border-radius: 20px !important;
+		border-radius: 12px !important;
```

Comment:
I know the `!important` bit isn't an addition from this PR but I see a few of these declarations throughout this file. Is there a way to add a little more specificity and avoid using `!important`?

Reply:
Good catch, it seems it wasn't really necessary, I've just remove it 😄